lib/autoyast.pm

```diff
@@ -713,9 +713,13 @@ sub upload_profile {
     my $profile = $args{profile};
     my $path = $args{path};
 
-    if (check_var('IPXE', '1')) {
-        $path = get_required_var('SUT_IP') . $path;
```

Comment:
We should agree with @mdoucha if we want to keep this or not, or maybe add another variable to control this?

Reply:
I see it is moved to another place in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17583. I'll not leave this part to your team as you are working on it. In my a few verify tests, these lines are not required at here.

Reply:
I keep this change in this PR for a moment for sake of verifying tests on openqa.